docs/chronicle/rancher.md

```diff
@@ -0,0 +1,120 @@
+# Chronicle SUSE Rancher Cookbook
+
+Recipe for installing Chronicle from the SUSE Rancher marketplace
+
+## Useful Links
+
+* [Chronicle Overview](https://btp.works/chronicle)
+* [Chronicle Docs](https://docs.btp.works/chronicle)
+* [Chronicle Examples](https://examples.btp.works)
+
+## Prerequisites
+
+You will need the following:
+
+!!!Important
+    As Chronicle uses Sawtooth as it backing ledger, a minimum of 4 nodes is
+    required for deployment.
+
+* 4 node Kubernetes cluster managed by
+  [SUSE Rancher](https://www.suse.com/products/suse-rancher/) v2.6 or later.
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) configured to
+  access your cluster
+
+## Install Chronicle
+
+Log in to Rancher and select the cluster you want to install Chronicle on.
+In our example, this will be the local cluster
+
+From the left menu, select _Apps_ and then _Charts_.
+Choose the Chronicle chart from the list of partner charts:
+
+![Partner Charts](../images/chronicle/rancher/partner-charts.png)
+
+This will take you to the following screen,
+click on the _Install_ button on the top right of the page:
+
+![Chart Details](../images/chronicle/rancher/chart-details.png)
+
+Here, you will need to specify the _namespace_ for your Chronicle
+installation. In our example, we will use the existing `chronicle` namespace.
+
+Now, click the _Next_ button on the bottom right of the page:
+
+![Install Namespace](../images/chronicle/rancher/install-namespace.png)
+
+On this screen you can configure your Chronicle installation. On the left hand
+side, you will find three options:
+
+* **Chronicle Settings** - Here you can configure the Chronicle image and tag
```

Comment:
As discussed we need a link here to explain how the generic default can be replaced - at a minimum - by one of our examples if not their own

Reply:
Therefore we need to include a link that points at a how to do this and better still make the default one of the examples eg manufacturing

Reply:
Since no where do we talk about a generic chronicle server